pydarn/plotting/fan.py

```diff
@@ -177,15 +178,14 @@ def plot_fan(cls, dmap_data: List[dict], ax=None, ranges: List=[],
                                                          scan_time)
         # Locate scan in loaded data
         plot_beams = np.where(beam_scan == scan_index)
-        hemisphere = SuperDARNRadars.radars[dmap_data[0]['stid']].hemisphere
```

Comment:
I can't plot southern hemisphere data any more, it plots the northern hemisphere with nothing on it. This change and the kwargs['hemisphere'] change below could be the culprit.